src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .map(string => string.trim())
+    .filter(string => string);
+
+  const stylesObject = stylesArray.reduce((acc, string) => {
```

Comment:
Why would you want to create a variable for a value that you will return right away?
```suggestion
  return stylesArray.reduce((acc, string) => {
```